client/src/components/ButtonCancel.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Button } from 'primereact/button';
+
+export function ButtonCancel(props) {
+  return (
+    <Button
+      type="button"
+      onClick={props.handleClickCancel}
```

Comment:
Conventionally, the function that triggers the event should be called `on<SomeEventHere>` and the function that responds to the event should be called `handle<SomeEventHere>`

this should be `onClick={props.onClick}` so that you can use it like

```
import { ButtonCancel } from './ButtonCancel'
 
class MyPage {
    render() {
       return (
          <ButtonCancel
              onClick={this.handleClickCancel}
           />
        )
     }

     handleClickCancel = () => {
        // do something
     }
}
```